packages/datatrak-web/src/utils/date.ts

```diff
@@ -3,8 +3,6 @@
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import moment from 'moment';
-
 export const shortDate = (date: Date) => {
```

Comment:
Good call getting rid of moment! Can we please rename this util to date display? All the dates seem to be the short format for now so it is clearer to give it a more obvious name.